CUBRIDdb/cursors.py

```diff
@@ -45,51 +129,106 @@ def close(self):
         self._cs.close()
         self._cs = None
 
-    def _bind_params(self, args,set_type=None):
+    def _prepare(self, query):
+        if not isinstance(query, (bytes, bytearray)):
+            stmt = query.encode(self.charset)
+        else:
+            stmt = query
+
+        if sys.version_info >= (3, 0):
+            stmt = stmt.decode()
+
+        self._cs.prepare(stmt)
+
+    def _bind_params(self, args, set_type=None):
+        """
+        Bind parameters to a command statement in a database cursor.
+
+        This method processes the provided arguments (args) and binds them to a command
+        statement associated with the database cursor. It handles different types of
+        arguments including booleans, iterables, bytes, and other data types by converting
+        or processing them appropriately before binding.
+
+        For each argument in 'args':
+        - If the argument is None, it is skipped.
+        - If the argument is a boolean, it is converted to '1' or '0' string.
+        - If the argument is an iterable (except strings and bytes), its element type
+        is determined using 'get_set_element_type', and then it's bound as a set.
+        - If the argument is a bytes object, it is converted to a binary string using
+        'bytes_to_binstr' and bound with type 'field_type.VARBIT'.
```

Comment:
```suggestion
        'bytes_to_binary_string' and bound with type 'FIELD_TYPE.VARBIT'.
```
다른 곳에도 `field_type.VARBIT` 과 같이 소문자 사용이 혼재되어 있는데 모두 대문자로 통일하는 것은 어떤가요?